src/api.ts

```diff
@@ -482,10 +483,21 @@ const tokenEndpoint = async (req: FastifyRequest, reply: FastifyReply) => {
     } catch (e) {
         const error = e as TokenError
         console.error('token endpoint fault',error)
+        setSessionCookie(reply, '')
```

Comment:
Clearing the session cookie on every token error here in an attempt to prevent the FE and AS from getting out of sync.